### DISCUSSION TITLE
Can't get attribute 'Accumulator' on module 'pytorch_lightning.trainer.supporters'

### DISCUSSION THREAD

**Discussion by sun-peach** [Lightning-AI/pytorch-lightning#6856] [General]
My version is 1.3.0.dev0 (I pip install from the master branch of git). However, when I run model.load_from_checkpoint(ckpt_path), it failed and shows the problem.

When I run on another PC, everything is good. Does anyone encounter this issue?

Thank you.

**Comment by ananthsub** [comment]
Could you post the full stack trace you're seeing?
